playwright-e2e/pages/start-application.ts

```diff
@@ -12,6 +17,10 @@ export class StartApplication {
   readonly hearingUrgencyHeader: Locator;
   readonly groundsForTheApplicationHeading: Locator;
   readonly groundsForTheApplicationHasBeenUpdatedFinished: Locator;
+  readonly 
```

Comment:
this might be mistakenly added, remove readonly from line 20?

Reply:
true